server/src/server/wrapper/model/error.ts

```diff
@@ -5,3 +5,11 @@ export class CcloudSessionExpiredException extends Error {
     this.message = "Login session has expired";
   }
 }
+
+export class CcloudTopicAlreadyExistsException extends Error  {
```

Comment:
Do we need to indicate that the error stems from ccloud?

Reply:
We don't _need_ to. 275a6bb0